<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test</class>
 <widget class="QWidget" name="Test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>640</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QStackedWidget" name="stacked_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="loading_page">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="loading_gif_label">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="result_page">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="distance_area">
         <item>
          <spacer name="distance_area_top_spacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="distance_chart_widget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>300</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="threshold_title_area">
           <item>
            <widget class="QLabel" name="threshold_title_label">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #3e3e3e;</string>
             </property>
             <property name="text">
              <string>閾値</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="threshold_label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #3e3e3e</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="about_threshold_button">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;閾値とは&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;AIが&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#3fda68;&quot;&gt;良品&lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;と&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#e66643;&quot;&gt;不良品&lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;を判別する境界です。この値を変更すると、AIの性能が変化します。&lt;br/&gt;性能評価の&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#3fda68;&quot;&gt;正解率&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;が高く&lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;、&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#e66643;&quot;&gt;見逃し率&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;が低く&lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;なるよう調整するのをおすすめします。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="styleSheet">
              <string notr="true">border: none;
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../qrc/icon.qrc">
               <normaloff>:/icon/resources/base/fonts/fontawesome/font-awesome_4-7-0_question-circle_32_4_ffc10e_none.png</normaloff>:/icon/resources/base/fonts/fontawesome/font-awesome_4-7-0_question-circle_32_4_ffc10e_none.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="threshold_title_area_spacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QSlider" name="threshold_slider">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="value">
            <number>80</number>
           </property>
           <property name="tracking">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
           <property name="tickInterval">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="distance_area_bottom_spacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="performance_area">
         <item>
          <spacer name="performance_area_top_spacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="performance_title_label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #3e3e3e;</string>
           </property>
           <property name="text">
            <string>性能評価</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="performance_chart_area">
           <item>
            <spacer name="performance_chart_area_left_spacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QWidget" name="performance_chart_background" native="true">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>250</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: transparent</string>
             </property>
             <widget class="QWidget" name="performance_chart_widget" native="true">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>250</width>
                <height>250</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
             <widget class="QWidget" name="verticalLayoutWidget">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>80</y>
                <width>111</width>
                <height>98</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="performance_rates_area">
               <item>
                <layout class="QHBoxLayout" name="correct_rate_area">
                 <item>
                  <widget class="QLabel" name="correct_rate_title_label">
                   <property name="styleSheet">
                    <string notr="true">color: #3e3e3e</string>
                   </property>
                   <property name="text">
                    <string>正解率</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="correct_rate_area_spacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="correct_rate_label">
                   <property name="font">
                    <font>
                     <pointsize>15</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: #3FDA68</string>
                   </property>
                   <property name="text">
                    <string>97%</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="performance_rates_area_spacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <layout class="QHBoxLayout" name="false_positive_rate_area">
                 <item>
                  <widget class="QLabel" name="false_positive_rate_title_label">
                   <property name="styleSheet">
                    <string notr="true">color: #3e3e3e</string>
                   </property>
                   <property name="text">
                    <string>誤検知率</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="false_positive_rate_area_spacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="false_positive_rate_label">
                   <property name="font">
                    <font>
                     <pointsize>15</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: #FFA00E</string>
                   </property>
                   <property name="text">
                    <string>2%</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="false_negative_rate_area">
                 <item>
                  <widget class="QLabel" name="false_negative_rate_title_label">
                   <property name="styleSheet">
                    <string notr="true">color: #3e3e3e</string>
                   </property>
                   <property name="text">
                    <string>見逃し率</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="false_negative_rate_area_spacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="false_negative_rate_label">
                   <property name="font">
                    <font>
                     <pointsize>15</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: #E66643</string>
                   </property>
                   <property name="text">
                    <string>1%</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <spacer name="performance_chart_area_right_spacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="performance_area_bottom_spacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qrc/icon.qrc"/>
 </resources>
 <connections/>
</ui>
